workflow-service-sdk/README.md

```diff
@@ -12,7 +12,7 @@ This is the API documentation for the Parodos Workflow Service. It provides oper
 ## Requirements
 
 Building the API client library requires:
-1. Java 1.7+
+1. Java 1.8+
```

Comment:
Aren't we using Java 11 ?

Reply:
We are using Java 17
This is auto generated README for the SDK requirements

Reply:
You're right! Thanks for the clarification!